mp/src/game/shared/momentum/mom_stickybomb.h

```diff
@@ -40,6 +40,8 @@ class CMomStickybomb : public CMomExplosive
     void Fizzle();
     void Detonate();
     void VPhysicsCollision(int index, gamevcollisionevent_t *pEvent) OVERRIDE;
+
+    bool GetUseImpactNormal();
```

Comment:
Two things:

1. Not a fan of the `UseImpactNormal` name for this variable. If all this variable is used for is to denote that the sticky hit the world/something, this should be renamed to `DidHitWorld`. Apply this to the `m_bUseImpactNormal` guy as well: `m_bHitWorld`

2. If this is just returning a member bool, you can inline it in the header and make it const:
```suggestion

    bool DidHitWorld() const { return m_bDidHitWorld; }
```